### DISCUSSION TITLE
Daisy UI Modal Focus

### DISCUSSION THREAD

**Discussion by Degreez90** [saadeghi/daisyui#2171] [Q&A] [answered]
I'm having issues with daisy UI Modal component. Whenever I click the button to open the Modal it auto focuses 
```
<button className='btn btn-sm btn-circle btn-ghost absolute right-2 top-2'>
              ✕
            </button> 
```
After the modal has popped up. This is without pressing Tab.
If I remove the button it will focus the next thing such as: 
```
<NavLink>
                  example 
                  </NavLink>
```
Is this by design?
Are there anyways of preventing this?

**Comment by saadeghi** [comment, marked as answer]
See this https://github.com/saadeghi/daisyui/issues/1893#issuecomment-1573647893
